omod/src/main/webapp/resources/htmlforms/sierra_leone/vitals.xml

```diff
@@ -549,9 +549,47 @@
             </fieldset>
 
         </includeIf>
+<fieldset>
+	<p>
+            <legend><uimessage code="mirebalais.vitals.response.title"/></legend>
+<!--taken from haiti/hiv/isanteplus VisiteDeSuivi.xml -->
+            <h3><uimessage code="mirebalais.vitals.response.title"/></h3>
+<table width="100%">
+		<tr>
+			<td width="50%" valign="TOP">
+				<section> 
+<!--					 <table>
```

Comment:
Committing commented-out code is not great. Plus, I assume this was copy-pasted from somewhere, so it's not like we're going to lose it forever. I'd just delete it this commented-out block.